dotCMS/src/main/java/com/dotmarketing/quartz/job/DeleteFieldJob.java

```diff
@@ -172,6 +173,7 @@ public static void triggerDeleteFieldJob(final Structure contentType, final Fiel
 	 *             An error occurred when deleting the Field.
 	 */
 	@Override
+    @CloseDBIfOpened
```

Comment:
Let's leave this one as it is for now. We can revisit later as part of the other task we have for this